sky/serve/constants.py

```diff
@@ -69,7 +69,7 @@
 # automatically generated from this start port.
 CONTROLLER_PORT_START = 20001
 LOAD_BALANCER_PORT_START = 30001
-LOAD_BALANCER_PORT_RANGE = '30001-30100'
+LOAD_BALANCER_PORT_RANGE = '30001-30020'
```

Comment:
Any reason we reduce the range of the LB ports?

Reply:
Opening many ports in LB mode can be expensive on hosted Kubernetes (GKE/EKS) can be expensive since each will create a load balancer service. On on-prem k8s it may exhaust list of public IPs available on the cluster. Also IIRC creating more than 100 services on GKE requires some quota extension..